dataprep/data_connector/connector.py

```diff
@@ -136,6 +162,98 @@ def _fetch(
 
         return resp
 
+    def query(
+        self,
+        table: str,
+        _auth: Optional[Dict[str, Any]] = None,
+        _count: Optional[int] = None,
+        **where: Any,
+    ) -> pd.DataFrame:
+        """
+        Query the API to get a table.
+
+        Parameters
+        ----------
+        table : str
+            The table name.
+        _auth : Optional[Dict[str, Any]] = None
+            The parameters for authentication. Usually the authentication parameters
+            should be defined when instantiating the Connector. In case some tables have different
+            authentication options, a different authentication parameter can be defined here.
+            This parameter will override the one from Connector if passed.
+        _count: Optional[int] = None,
+            count of returned records.
+        **where: Dict[str, Any]
+            The additional parameters required for the query.
+
+        Returns
```

Comment:
No need for the Returns statement. Check our wiki for docstring best practice https://github.com/sfu-db/dataprep/wiki/Docstring-Best-Practice